packages/modules/viewer-react/src/components/app-ui/providers/ViewerNavigationToolsProvider.tsx

```diff
@@ -30,13 +29,13 @@ export class ViewerNavigationToolsProvider extends StandardNavigationToolsProvid
     });
   }
 
-  public override provideToolbarButtonItems(
+  public provideToolbarButtonItems(
```

Comment:
Because the function changed name :) So the solution is not to remove the overrride, but to rename the function. In any case, the override should be on the `provideToolbarItemsInternal` instead to benefit from the overrides, however if you are using it directly and not "providing" this to anyone, it doesnt make much of a difference.

Reply:
changed the name of the function to provideToolbarItemsInternal and added back the override

Reply:
this still smells fishy; i need to review what happened here in the appui level